### PR TITLE
Events

### PR DESCRIPTION
**Pull request by sajjadrajaee**
Add calculate.js
Add operate.js
Add Keys.js
add events and lifting-state up.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,43 +1,37 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 import '../style.css';
+import KeyBtn from './Kyes';
 // eslint-disable-next-line react/prefer-stateless-function
```
Comment (FrederickMih) [inline src/components/Calculator.js:5]: -  We are not supposed to disable Linters. Linters monitor your code and flag programmatic/stylistic errors. Always try to fix them.

**Review by FrederickMih — APPROVED** [review]
Hi @sajjadrajaee , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Correct Gitflow :heavy_check_mark: 
- No Linter error :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (sajjadrajaee) [reply src/components/Calculator.js:5]: Thank you to review my codes. I searched a lot but I can't find any solution for this type of error. can you help me to fix this linter without disabling it? 
thank you in advance.

**Review by bobb-Rob — COMMENTED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @sajjadrajaee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**